README.md

```diff
@@ -1,110 +1,162 @@
+You will need to run `npm run build` again if changes are made to:
+* the [govuk-frontend](https://github.com/alphagov/govuk-frontend) library
+* translations
+* images
 
 ### Configuring core-back to work with a local core-front
-In the core common infra repository dev-deploy documentation there are instructions on how to configure your dev
-stack to work with a locally running core-front.
 
-### Code Owners
+You can use the [dev-deploy tool](https://github.com/govuk-one-login/ipv-core-common-infra/blob/main/utils/dev-deploy/README.md) to [configure your core-back to work with a local core-front](https://github.com/govuk-one-login/ipv-core-common-infra/blob/main/utils/dev-deploy/docs/local-development.md#configuring-core-back-to-work-with-a-local-core-front).
 
-This repo has a `CODEOWNERS` file in the root and is configured to require PRs to reviewed by Code Owners.
+## Using pre-commit to verify your commits
 
-## Pre-Commit Checking / Verification
+We use the [pre-commit](https://pre-commit.com/) tool to identify issues before you commit your code. It uses Git hook scripts which can be configured in [`.pre-commit-config.yaml`](https://github.com/govuk-one-login/ipv-core-front/blob/main/.pre-commit-config.yaml).
 
-There is a `.pre-commit-config.yaml` configuration setup in this repo, this uses [pre-commit](https://pre-commit.com/) to verify your commit before actually committing, it runs the following checks:
+On running `git commit`, pre-commit Git hooks check for:
 
-- Check Json files for formatting issues
-- Fixes end of file issues (it will auto correct if it spots an issue - you will need to run the git commit again after it has fixed the issue)
-- It automatically removes trailing whitespaces (again will need to run commit again after it detects and fixes the issue)
-- Detects aws credentials or private keys accidentally added to the repo
-- runs cloud formation linter and detects issues
-- runs checkov and checks for any issues
-- runs detect-secrets to check for secrets accidentally added - where these are false positives, the `.secrets.baseline` file should be updated by running `detect-secrets scan > .secrets.baseline`
+- formatting issues in JSON files
+- end of file issues
+- trailing whitespaces
+- AWS credentials or private keys that have been accidentally added
+- AWS CloudFormation issues
+- infrastructure issues by running [checkov](https://github.com/bridgecrewio/checkov) 
+- secrets that have been accidentally added by running [detect-secrets](https://github.com/Yelp/detect-secrets) - if these are false positives, update the `.secrets.baseline` file by running `detect-secrets scan > .secrets.baseline`
 
-### Dependency Installation
+### Committing after automatic changes
 
-To use this locally you will first need to install the dependencies, this can be done in 2 ways:
+End of file issues and trailing whitespaces are automatically fixed. If this happens, run `git commit` again to commit the changes.
```

Comment:
```suggestion
Pre-commit automatically fixes end of file issues and trailing whitespaces. If this happens, run `git commit` again to commit the changes.
```
Just a suggestion to avoid passive voice.